domain/BUILD.bazel

```diff
@@ -102,6 +102,96 @@ domain_test(
     deps = TEST_DEPS,
 )
 
+domain_test(
```

Comment:
FYI that #2240 was just submitted. You'll probably get merge conflicts from this--can you update your changes with the latest develop?

Reply:
Done